packages/renderer-react/src/index.js

```diff
@@ -1,6 +1,5 @@
 import React from 'react';
 import { render, unmountComponentAtNode } from 'react-dom';
-import { withRenderer } from 'skatejs';
```

Comment:
This `skatejs` didn't seem to be used in this module, so I decided to remove it, along with the dependency on `skatejs` from `package.json`